repository/src/main/java/uk/gov/hmcts/ccd/definition/store/repository/entity/CaseTypeEntity.java

```diff
@@ -88,6 +88,11 @@
     @JoinColumn(name = "case_type_id")
     private final List<CaseTypeUserRoleEntity> caseTypeUserRoleEntities = new ArrayList<>();
 
+    @OneToMany(fetch = EAGER, cascade = ALL, orphanRemoval = true)
```

Comment:
I don't believe this should be `EAGER`. When the case type is read from the API, the case roles are not needed and should not be retrieved. Making that relation eager is only going to decrease the performance when reading the case type from the DB.

Reply:
ok